.github/workflows/deploy.yml

```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
   deploy:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```

Comment:
@yannikschaelte Do you know if `ubuntu-latest` is now possible?

Reply:
yes, that should work (I thought I had changed it to latest everywhere)